lib/omnibus/packagers/solaris.rb

```diff
@@ -35,11 +35,10 @@ class Packager::Solaris < Packager::Base
       copy_file("#{project.package_scripts_path}/postinst", '/tmp/pkgmk/postinstall')
       copy_file("#{project.package_scripts_path}/postrm", '/tmp/pkgmk/postremove')
 
-      Dir.chdir(staging_dir) do
```

Comment:
We don't need to chdir to staging_dir here. All commands pkgmk, pkgchk and pkgtrans take full paths as arguments.